From: Peter Xu <peterx@redhat.com>
Subject: selftests-mm-use-test_gen_progs-where-proper-fix
Date: Sun, 16 Apr 2023 17:35:02 -0400

tools/testing/selftests/mm/Makefile: don't wipe out TEST_GEN_PROGS

Link: https://lkml.kernel.org/r/ZDxrvZh/cw357D8P@x1n
Signed-off-by: Peter Xu <peterx@redhat.com>
Cc: Axel Rasmussen <axelrasmussen@google.com>
Cc: David Hildenbrand <david@redhat.com>
Cc: Dmitry Safonov <0x7f454c46@gmail.com>
Cc: Mike Kravetz <mike.kravetz@oracle.com>
Cc: Mike Rapoport (IBM) <rppt@kernel.org>
Cc: Zach O'Keefe <zokeefe@google.com>
Signed-off-by: Andrew Morton <akpm@linux-foundation.org>
---

 tools/testing/selftests/mm/Makefile |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

--- a/tools/testing/selftests/mm/Makefile~selftests-mm-use-test_gen_progs-where-proper-fix
+++ a/tools/testing/selftests/mm/Makefile
@@ -42,7 +42,7 @@ TEST_GEN_PROGS += hugepage-mremap
 TEST_GEN_PROGS += hugepage-shm
 TEST_GEN_PROGS += hugepage-vmemmap
 TEST_GEN_PROGS += khugepaged
-TEST_GEN_PROGS = madv_populate
+TEST_GEN_PROGS += madv_populate
 TEST_GEN_PROGS += map_fixed_noreplace
 TEST_GEN_PROGS += map_hugetlb
 TEST_GEN_PROGS += map_populate
_
